test/pipeK.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var S = require('..');
+
+var eq = require('./internal/eq');
+
+
+test('pipeK', function() {
+
+  eq(typeof S.pipeK, 'function');
+  eq(S.pipeK.length, 2);
+  eq(S.pipeK.toString(), 'pipeK :: (Foldable f, Chain m) => f (Any -> m Any) -> m a -> m b');
+
+  eq(S.pipeK([], S.Just([1, 2, 3])), S.Just([1, 2, 3]));
+  eq(S.pipeK([S.tail], S.Just([1, 2, 3])), S.Just([2, 3]));
+  eq(S.pipeK([S.tail, S.tail], S.Just([1, 2, 3])), S.Just([3]));
+  eq(S.pipeK([S.tail, S.tail, S.head], S.Just([1, 2, 3])), S.Just(3));
+
+});
```

Comment:
Let's add a block of tests for `List (Any -> Maybe Any)` much as we did for `pipe`.

Reply:
⚡️